lib/state.js

```diff
@@ -95,6 +95,10 @@ StateResolver.prototype.capture = function() {
   // Gather the stack frames first
   var that = this;
   var frames = that.resolveFrames_();
+  this.nativeErrorMirror =
+    this.state_.frame(0).evaluate('\'Cannot display native properties.\'');
+  this.getterErrorMirror =
+    this.state_.frame(0).evaluate('\'Cannot display properties with getters.\'');
```

Comment:
It would be preferable to return a `StatusMessage` indicating error rather than a string. See how the buffer full error message works:

```js
  // The 0-th object in the variable table is a sentinel 'buffer full' error
  // message value
  this.rawVariableTable_ = [ null ];
  this.resolvedVariableTable_ = [
    { status: new StatusMessage(StatusMessage.VARIABLE_VALUE,
                                'Buffer full', true) }
```